use-timescale/migration/postgres-pg-dump-and-restore.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate from Postgres using pg_dump and pg_restore
+excerpt: Migrate a entire database with native PostgreSQL commands
+products: [cloud, self_hosted]
+keywords: [dump, restore]
+tags: [recovery, pg_dump, pg_restore]
+---
+
+import ConsiderCloud from "versionContent/_partials/_consider-cloud.mdx";
+
+# Migrate to Timescale Cloud using `pg_dump` and `pg_restore`
+
+You can backup and restore an entire database using
+the native PostgreSQL [`pg_dump`][pg_dump] and [`pg_restore`][pg_restore]
+commands.
+
+## Back up your entire database
+
+You can perform a backup using the `pg_dump` command at the command prompt. For
+example, to backup a database named `tsdb`:
+
+```bash
+pg_dump -Fc -f tsdb.bak tsdb
```

Comment:
We are recommending customers to use the flags `--no-tablespaces --no-owner --no-privileges` to prevent permission errors.